sink/datadog.go

```diff
@@ -487,6 +499,25 @@ func (s *Datadog) estimateSize(metrics []datadogV2.MetricSeries) (int, error) {
 	return size / len(metrics), nil
 }
 
+func (s *Datadog) uniqIdentifierForMetric(name string, tags []string) string {
+	var key string
+	key += name
+	for _, t := range tags {
+		v := s.intForTag(t)
+		key += strconv.Itoa(int(v))
+	}
+	return key
+}
```

Comment:
Are we concerned about tag values possibly being inconsistently ordered? We may want to consider sorting the tags prior to generating the key to ensure a consistent order.

Reply:
I had that suspicion as well, although the code flow kind of indicate it should be consistently ordered but it's probably not worth the risk. It will add a bit more overhead on for every metric on every send. Should be alright. I will add it.